src/app/screens/LNURLPay.tsx

```diff
@@ -40,11 +40,11 @@ function LNURLPay(props: Props) {
         JSON.parse(searchParams.get("origin") as string)) ||
       getOriginData()
   );
-  const [valueMSat, setValueMSat] = useState<number | undefined>(
-    details?.minSendable
+  const [valueSat, setValueSat] = useState(
+    (details?.minSendable && (+details?.minSendable / 1000).toString()) || ""
```

Comment:
Here you're using `+` instead of `parseInt`. Would one way work everywhere?

Reply:
Here we treat it as raw input and just coerce to number (if it was a string), leaving decimals intact. parseInt always expects a string which in this case could be a number already.